scripts/bootstrapCatalogWithSam.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I think the name "bootstrap" is confusing here, also I think the one bash script we have now uses `_` instead of camel case so this filename should follow that convention. Also "catalog" seems redundant since we aren't going to have other sam initialization scripts in this repo. Maybe `create_groups_in_sam.sh`?

Reply:
I agree.